pkg/kubelet/kubelet.go

```diff
@@ -339,6 +399,32 @@ func (kl *Kubelet) setupDataDirs() error {
 	return nil
 }
 
+func (kl *Kubelet) setupTmpfsDataDirs() error {
+	kl.tmpfsRootDirectory = path.Clean(kl.tmpfsRootDirectory)
+	// TODO: handle edge cases like mount already exists
```

Comment:
@thockin @markturansky I think `linux` would be a fine build-flag for the mount stuff personally.  Unless I'm mistaken I don't think we have any !linuxes in mind for platforms to run a kubelet on.